pkg/policy/rule_l4.go

```diff
@@ -197,6 +216,25 @@ func (l4 L4PolicyMap) HasRedirect() bool {
 	return false
 }
 
+// containsAllL4 checks if the L4PolicyMap contains all `l4Ports`. Returns false
+// if the `L4PolicyMap` has a single rule and l4Ports is empty or if a single
+// `l4Port`'s port is not present in the `L4PolicyMap`.
+func (l4 L4PolicyMap) containsAllL4(l4Ports []*models.Port) bool {
+	if len(l4) == 0 {
+		return true
+	}
+	if len(l4Ports) == 0 {
+		return false
```

Comment:
I should probably change the name of the `containsAllL4` to a different one, suggestions needed.
The logic here is, we have an L4PolicyMap loaded, if the `l4Ports` is empty this means the L4 policy will be denied for that search context. Makes sense?